boards/arm/stm32h747i_disco/stm32h747i_disco_m4.dts

```diff
@@ -53,6 +53,9 @@
 
 &uart8 {
 	status = "okay";
+        pinctrl-0 = <&uart8_tx_pj8 &uart8_rx_pj9>;
```

Comment:
definition exists in the boards/arm/stm32h747i_disco/stm32h747i_disco.dtsi

Reply:
I see it. Will remove